src/helm/proxy/clients/auto_client.py

```diff
@@ -133,7 +133,7 @@ def _get_client(self, model: str) -> Client:
                 from helm.proxy.clients.google_client import GoogleClient
 
                 client = GoogleClient(cache_config=cache_config)
-            elif organization in ["together", "databricks", "meta", "stabilityai"]:
+            elif organization in ["together", "databricks", "EleutherAI", "meta", "stabilityai"]:
```

Comment:
I think we should decide on style - currently, org is all lowercase (e.g., stabilityai), so we might want to keep that?

Reply:
Sure, I changed the org name to lowercase. Note:

- `AlephAlpha` is still title cased
- The tokenizer name is still spelled with `EleutherAI`

Reply:
Let's do lowercase for now.